feature_engine/outliers/winsorizer.py

```diff
@@ -95,6 +200,11 @@ class Winsorizer(BaseOutlier):
         both sides. Thus, when `capping_method='quantiles'`, then `'fold'` takes values
         between 0 and 0.20.
 
+    add_indicators: bool, default=False
+        Whether to add indicator variables indicating in each variable was capped.
+        If passing 'True', will output these with the same names as the input
+        variables plus suffizes '_low' and '_high'.
+
```

Comment:
Could we re-phrase these lines to say something like this:

Whether to add indicator variables to flag the capped outliers.
If 'True', binary variables will be added to flag outliers on the left and right 
tail of the distribution. One binary variable per tail, per variable.